test/collection.js

```diff
@@ -1355,6 +1355,14 @@
     ok(collection.comparator === comparator);
   });
 
+  test("Pass `null` for `models` for empty Col with `options`", 2, function() {
```

Comment:
I've added `false`, `undefined` and `0`. Let me know if `0` is overdoing it a bit.